src/site/content/en/blog/centering-in-css/index.md

```diff
@@ -0,0 +1,347 @@
+5. **Flow**: great!
+
+It's going to be hard to beat the conciseness of `display: grid` and the
+`place-content` shorthand. Since it centers and justifies children collectively,
+it's a solid centering technique for groups of elements meant to be read.
+
+```css
+.content-center {
+  display: grid;
+  place-content: center;
+  gap: 1ch;
+}
+```
+
+<div class="w-columns">
+{% Compare 'worse', 'Pros' %}
+- Content is centered even under constrained space and overflow  
+- Centering edits and maintenance are all in one spot  
+- Gap guarantees equal spacing amongst _n_ children  
+- Grid creates rows by default  
+
+{% endCompare %}
+
+{% Compare 'better', 'Cons' %}
+- The widest child (`max-content`) sets the width for all the rest. This will be
+  discussed more in [Gentle Flex](#gentle-flex).  
+
+{% endCompare %}
+</div>
+
+**Great for** macro layouts containing paragraphs and headlines, prototypes, or
+generally things that need legible centering.
+
+{% Aside %} `place-content` is not exclusive to `display: grid`. `display: flex`
+can also benefit from `place-content` and `place-item`
+[shorthands](https://codepen.io/argyleink/pen/PoqWOPZ). {% endAside %}
+
+### Gentle Flex
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/gentle-flex-ringer-cycle.mp4">
+  </video>
+</figure> 
+
+1. **Squish:** great!
+2. **Squash:** great!
+3. **Duplicate:** great!
+4. **Edit:** great!
+5. **Flow:** great!
+
+Gentle Flex is a truer centering-*only* strategy. It's soft or gentle, because
+unlike `place-content: center`, no children's box sizes are changed during the
+centering. As gently as possible, all items are stacked, centered, and spaced. 
+
+```css
+.gentle-flex {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1ch;
+}
+```
+
+<div class="w-columns">
+{% Compare 'worse', 'Pros' %}
+- Only handles alignment, direction, and distribution
+- Edits and maintenance are all in one spot
+- Gap guarantees equal spacing amongst _n_ children
+
+{% endCompare %}
+
+{% Compare 'better', 'Cons' %}
+- 5 lines of code
+
+{% endCompare %}
+</div>
+
+**Great for** both macro and micro layouts.
+
+{% Aside "key-term" %} Macro layouts are like states or territories of a
+  country, very high level, large coverage zones. The zones created by macro
+  layouts tend to contain more layouts. The less surface the layout covers, the
+  less of a macro layout it becomes. As a layout covers less surface area or
+  contains less layouts, it becomes more of a micro layout. {% endAside %}
+
+### Autobot
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/autobot-ringer-cycle.mp4">
+  </video>
+</figure> 
```

Comment:
Typo in video (`contnet`)

![image](https://user-images.githubusercontent.com/4713486/91222541-33848700-e6d4-11ea-8977-b042b6ec2b63.png)

Reply:
each video kinda has charming, humanizing, and messy typos and typing in it that i felt brought it character. would i need to shoot each video perfectly to pass? the typos are part of the chaos are part of the fun 🙂

Reply:
OK I understand your rationale. I worry that some people will see it as unprofessional but I'm OK with keeping it.